ext/DataFramesExt.jl

```diff
@@ -0,0 +1,10 @@
+module DataFramesExt
+
+using PlotlyJS
+isdefined(Base, :get_extension) ? (using DataFrames) : (using ..DataFrames)
+isdefined(Base, :get_extension) ? (using CSV) : (using ..CSV)
+
+
+dataset(::Type{DataFrames.DataFrame}, name::String) = DataFrames.DataFrame(PlotlyJS.dataset(CSV.File, name))
```

Comment:
Should extend the parent module's `dataset`, as in `ext/CSVExt.jl`, I think:
```suggestion
PlotlyJS.dataset(::Type{DataFrames.DataFrame}, name::String) = DataFrames.DataFrame(PlotlyJS.dataset(CSV.File, name))
```

Reply:
Good catch, thanks!